### DISCUSSION TITLE
Extensions/Modules still available for OpenMage

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#1955] [Extensions & Modules]
Some developers have chosen to eliminate the Magento 1 product offering but there are still quite a few that provide free or paid Magento 1 extensions/modules.

This post aims to provide information about these developers. Besides the link please be free to mention what products you are still using.

If you are a developer I don't see a bad thing in doing a little advertising here too.

**Comment by f1-outsourcing** [comment]


**Reply by addison74** [reply]
@f1-outsourcing - it would be better to delete your post so that it has nothing to do with the subject. I understand that you have identified real problems but here we are talking about extensions/modules and their developers. I expect those who post to tell us that X, Y and Z still have products available.

**Comment by elidrissidev** [comment]
Shameless plug: I've just finished my [first Magento extension](https://github.com/OpenMage/magento-lts/discussions/1986) last week, posting here for more people to notice because I think many will find it useful. Oh, and we're gonna be using it in my company soon, so it comes with active support too ;)

**Comment by f1-outsourcing** [comment]
Is there something like a centralised repo for modules that were opensource and free? (Currently looking for the delete orders extension/module)

**Reply by elidrissidev** [reply]
There is [OpenMageModuleFostering](https://github.com/OpenMageModuleFostering), and [awesome-openmage](https://github.com/fballiano/awesome-openmage).

**Reply by f1-outsourcing** [reply]
Thanks!!!
